ome_zarr/writer.py

```diff
@@ -634,18 +764,38 @@ def write_labels(
     :param label_metadata:
       Image label metadata. See :meth:`write_label_metadata` for details
     """
-    mip, axes = _create_mip(labels, fmt, scaler, axes)
-    write_multiscale_labels(
-        mip,
-        group,
+    sub_group = group.require_group(f"labels/{name}")
+    if isinstance(labels, da.Array):
+        _write_dask_image(
+            labels,
+            sub_group,
+            scaler,
+            chunks=chunks,
+            fmt=fmt,
+            axes=axes,
+            coordinate_transformations=coordinate_transformations,
+            storage_options=storage_options,
+            name=name,
+            **metadata,
+        )
+    else:
+        mip, axes = _create_mip(labels, fmt, scaler, axes)
+        write_multiscale(
```

Comment:
Does that effectively mean that `write_multiscales_label` should be deprecated or only that it applies to a subset of cases (and does not support dask)?

Reply:
`write_multiscale_labels` was calling `write_multiscale` before this PR too.   So basically I'm just trying to preserve the pre-dask code path.